### ISSUE TITLE
Clarification for `pollOnStart` parameter

### ISSUE THREAD

**Issue by marcoancona** [amplitude/experiment-js-client#111]
Hi all, 
We have recently started with the Experiment SDK. I was expecting `pollOnStart: true` to instantiate a poll mechanism that would fetch the user flags and variants on a regular basis. Inspecting the network requests and the source code, however, we found that the flag definitions are fetched every 60 seconds, but the _values_ for the current users are not re-evaluated automatically as part of this process.

I wonder, what is the purpose of fetching the flags so often without re-evaluating the flag values? I expect the value to change more often, as this depends on user properties/events that might affect the target audience of a variant.

**Comment by bgiori** [comment]
You're right, pollOnStart only polls for flag configurations, it does not periodically fetch remote variants. 

pollOnStart is mostly useful for local evaluation, which is a less common use case on the client-side. Unless you're using local evaluation I would recommend setting `pollOnStart: false`. If you want to re-fetch remote evaluation variants at an interval you would need to implement that manually in your app using setInterval or something similar.

**Comment by marcoancona** [comment]
Thanks for clarifying @bgiori . Do you have a recommendation about how often it would be good to poll for this flag? Would it be a problem to fetch the flag values every minute, as `pollOnStart` does by default?
